pkg/cmd/root/root.go

```diff
@@ -46,17 +46,25 @@ func NewRootCommand(f *factory.Factory, version string) *cobra.Command {
 	var help bool
 	fs.BoolVarP(&help, "help", "h", false, f.Localizer.MustLocalize("root.cmd.flag.help.description"))
 
+	var devpreview *string = nil
+	cmd.Flags().StringVar(devpreview, "devpreview", "", f.Localizer.MustLocalize("root.cmd.flag.devpreview.description"))
```

Comment:
I spent most of the time thinking how to enable this properly. 
Choose string finally as the best way. 
Technically we can validate it for the proper values as well :)
This is as string because:

- boolean has only 2 possible values - we need 3
- It will simplify adding tiers later without introducing other flags etc.

Reply:
What will the valid values be?

Reply:
Open to your feedback.

Currently true, yes etc. but we can rename flqg and have alpha beta etc.

Reply:
https://github.com/spf13/cobra#persistent-flags

This should use persistent flags.

Reply:
The thing about string flags is the need an argument so you cannot just do `rhoas srs --devpreview` - its a bit more awkward to achieve.

Reply:
Acording to this issue boolean ones are not going to help us

https://github.com/spf13/cobra/issues/613

This doesn't work this way. Srs will not be visible. DevPreview flag on it's own modifies config so its not needed when calling dev preview commands. 
```
rhoas srs --devpreview
```